### ISSUE TITLE
Disable mute on click 

### ISSUE THREAD

**Issue by MisterBianco** [jaagr/polybar#1521]
## Describe the issue
I have spent the last hour looking for a way to disable the mute on click functionality of the volume and alsa modules. I don't need it to ever be muted via the bar so I would like to disable it while still maintaining the ability to click on the workspace icons.

**Expected behavior:**

I click on the volume modules bar and nothing happens, or if possible it moves the volume level to the approximate point in the bar, though I doubt that is possible.

**Actual behavior:**

It mutes my master volume and requires manual unmuting.

## To Reproduce
*A minimal but complete config with which the problem occurs:*
```dosini
[module/alsa]
    type = internal/alsa
    master-mixer = Master
    master-soundcard = hw:1
    ;click-left = ?
    interval = 5
    mapped = true

    format-volume = <bar-volume> <label-volume>
    label-muted = Muted
    label-muted-foreground = #689d6a
    label-volume-foreground = #689d6a

    bar-volume-width = 10
    bar-volume-foreground-0 = #aaff77
    bar-volume-foreground-1 = #aaff77
    bar-volume-foreground-2 = #fba922
    bar-volume-foreground-3 = #ff5555
    bar-volume-indicator = |
    bar-volume-indicator-foreground = #ff
    bar-volume-indicator-font = 3
    bar-volume-fill = ─
    bar-volume-fill-font = 3
    bar-volume-empty = ─
    bar-volume-empty-font = 3
    bar-volume-empty-foreground = #444444
```

## Environment:
* WM: i3-gaps
* Output of `polybar -vvv`:

polybar 3.1.0-161-g028b141

Features: +alsa +curl +i3 +mpd +network +pulseaudio +xkeyboard

X extensions: +randr (+monitors) -render -damage -sync -composite +xkb +xrm +xcursor

Build type: RelWithDebInfo
Compiler: /usr/bin/c++
Compiler flags:  -Wall -Wextra -Werror -Wno-noexcept-type -O2 -pedantic -pedantic-errors
Linker flags:

**Comment by patrick96** [comment]
There is no way to disable the built-in click actions of modules.

As for the unmuting issue, make sure you have set `speaker-mixer` correctly (and `headphone-mixer` if you use headphones). Polybar will mute/unmute the master and speaker mixers on click. If you don't specify `speaker-mixer`, polybar will mute only master, which also mutes most other mixers (this seems to be a pulse and alsa interaction issue: [bug report](https://bugs.launchpad.net/ubuntu/+source/pulseaudio/+bug/878986)). If you set `speaker-mixer` correctly, polybar will unmute that mixer as well fixing the issue.

I have openend an issue with a concept to clean up click actions in modules. If we implement it, it would also allow you to disable specific click actions. This is something I wanted to propose anyway and this gave me a reason.

**Comment by MisterBianco** [comment]
I love how responsive and forthcoming you all are. I appreciate that and look very much forward to this hopefully being implemented!

**Comment by l0f4r0** [comment]
Indeed, it would be nice to be able to disable specific default click-left|right|middle or even best to override them :)
